### PR TITLE
Test App

### PR DESCRIPTION
**Pull request by mohametalmeari**
# In this pull request we 
- [x] Tested Operate Functionality
- [x] Tested Calculate Functionality
- [x] Tested Components
- [x] Ran all tests

### CODE REVIEW

**Review by Ibizugbe — COMMENTED** [review]
Hi @mohametalmeari 👋👋👋, 

INVALID


While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have contributions from your learning partner🥲. Kindly note that this is a pair programing activity and you are required to work with your partner to complete this milestone. Please reach out to your partner and work with the person.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Layout.text.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { render } from '@testing-library/react';
+import Layout from './Layout';
+
+describe('Layout Component', () => {
+  test('test render of Layout', () => {
+    const { container } = render(<Layout />);
+    expect(container).toMatchSnapshot();
+  });
+});
```
Comment (julie-ify) [inline src/components/Layout.text.js:9]: - [ ] You made an error in this file's name. Kindly rename this file to `.test.`, instead of `.text.`

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @mohametalmeari and @tjay1760,

Good job so far!

## Highlights!
- Good code organization and tests ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- Jest snapshots used :heavy_check_mark: 
- User interactions tested with RTL :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mohametalmeari** [comment]
Thank you so much, happy weekend 🥂

**Pull request merged**